docs/HowTo/Monitor/Elastic-Stack.md

```diff
@@ -9,14 +9,15 @@ Elastic Stack (also known as ELK) is a common open-source log management platfor
 [Besu Sample Networks](https://github.com/PegaSysEng/besu-sample-networks) provides example
 implementations using Elastic Stack for log management.
 
-The [Filebeat configuration] ingests the logs and outputs them to Redis for storage. Redis provides
-a highly available mechanism enabling log storage by any of the Elastic Beats and pulled by
-Logstash as required.
+The [Filebeat] and [Metricbeat] configurations ingest the logs and output them to Redis for
```

Comment:
Suggest rename to `[Filebeat] configuration ingests logs, and the [Metricbeat] configuration collects metrics from the nodes at regular defined intervals,

Reply:
Done. Thanks!

Reply:
Done. Thanks!